src/app/components/ConditionalTooltip/ConditionalTooltip.tsx

```diff
@@ -0,0 +1,35 @@
+import { Tooltip, TooltipProps } from '@/components/Tooltip'
```

Comment:
I would prefer to have additional properties in the current button instead of creating a new component.

Reply:
why